src/VisualStudio/VisualBasic/Impl/Options/VisualBasicSettingsManagerOptionSerializer.vb

```diff
@@ -43,6 +43,7 @@ Namespace Microsoft.VisualStudio.LanguageServices.VisualBasic.Options
                                 New KeyValuePair(Of String, IOption)(SettingStorageRoot + "Outlining", FeatureOnOffOptions.Outlining),
                                 New KeyValuePair(Of String, IOption)(SettingStorageRoot + "EnableHighlightReferences", FeatureOnOffOptions.ReferenceHighlighting),
                                 New KeyValuePair(Of String, IOption)(SettingStorageRoot + "EnableHighlightRelatedKeywords", FeatureOnOffOptions.KeywordHighlighting),
+                                New KeyValuePair(Of String, IOption)(SettingStorageRoot + "RenameTrackingPreview", FeatureOnOffOptions.RenameTrackingPreview),
```

Comment:
@Pilchie - I double checked that these options persist across Vs sessions. It works because `FeatureOnOffOptions.OptionName` is passed into `ExportLanguageSpecificOptionSerializer` both here and for C#, and I had added `RenameTrackingPreview` to `FeatureOnOffOptions` because its really an On/Off option in this case. See the change in `FeatureOnOffOptions` [here](https://github.com/balajikris/roslyn/blob/AsyncPreviewRenameTracking/src/EditorFeatures/Core/Shared/Options/FeatureOnOffOptions.cs#L45)
Does this sound right?